src/renderer/components/MediaDisplayTranscribeProgress.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  styled,
+  Box,
+  Stack,
+  Typography,
+  CircularProgress,
+} from '@mui/material';
+import IconButton from '@mui/material/IconButton';
+import DoneIcon from '@mui/icons-material/Done';
+import ErrorIcon from '@mui/icons-material/Error';
+import CachedIcon from '@mui/icons-material/Cached';
+import colors from '../colors';
+
+enum AsyncState {
+  READY = 'READY',
+  LOADING = 'LOADING',
+  DONE = 'DONE',
+  ERROR = 'ERROR',
+}
+
```

Comment:
This enum is already defined in `RunTranscriptionView.tsx`. Hence, its appropriate to move it to `sharedTypes.ts`!